static/js/compose_closed_ui.js

```diff
@@ -119,9 +123,13 @@ export function initialize() {
         compose_actions.start("stream", {trigger: "new topic button"});
     });
 
-    $("body").on("click", ".compose_private_button", () => {
-        compose_actions.start("private", {trigger: "new private message"});
-    });
+    if (page_params.realm_private_message_policy === 2) {
+        $(".compose_private_button").attr("disabled", true);
+    } else if (page_params.realm_private_message_policy === 1) {
+        $("body").on("click", ".compose_private_button", () => {
+            compose_actions.start("private", {trigger: "new private message"});
+        });
+    }
```

Comment:
@timabbott what's our policy on code being future-proofed? if we add a new value, do we want the code to throw an error of some kind?

in `models.py` it looks like we're setting up for future values?
```
    PRIVATE_MESSAGE_POLICY_UNLIMITED = 1
    PRIVATE_MESSAGE_POLICY_DISABLED = 2
    private_message_policy: int = models.PositiveSmallIntegerField(
        default=PRIVATE_MESSAGE_POLICY_UNLIMITED
    )
    PRIVATE_MESSAGE_POLICY_TYPES = [
        PRIVATE_MESSAGE_POLICY_UNLIMITED,
        PRIVATE_MESSAGE_POLICY_DISABLED,
    ]
```